website/content/docs/concepts/security/permissions.mdx

```diff
@@ -36,61 +116,121 @@ level action infers being granted all subactions. Thus, if a grant conveys
+#### The `no-op` Action
 
 Starting in Boundary 0.2.1 there is an action that can be granted called
 `no-op`. As might be apparent, `no-op` is not used for any real action in
 Boundary; the purpose of this action is for listing visibility. Boundary only
 shows resources in the output of a `list` action for which a user has at least
 one granted action. Thus, without `no-op`, in order for a resource to be visible
-in a list, a different action such as `read` would have to be granted to a user.
-This could result in exposing more information than desired, especially in the
-case of listing scopes and authentication methods so that users can perform
-initial authentication to Boundary (that is, granting access to `u_anon`).
+in a `list` action, a different action such as `read` would have to be granted
+to a user. This could result in exposing more information than desired,
+especially in the case of listing scopes and authentication methods so that
+users can perform initial authentication to Boundary (that is, granting access
+to `u_anon`).
 
 By granting the `no-op` action to users, they can see the resources in the
 output of a list command without needing other capability grants as well.
 
-## Applicable Resource Types
+### Output Fields
 
-Boundary's [domain model](/docs/concepts/domain-model) is based on resource
-types. These can be implemented directly, such as with targets, or they can be
-abstract types that are implemented by concrete types within the system. As an
-example of the latter, a host catalog is an abstract type and a Static host
-catalog is a concrete type.
+Starting in Boundary 0.2.1, grant strings can contain an `output_fields` field.
+This allows fine-grained control over visibility of data returned to users.
 
-From a permissions standpoint, however, all actions take place against directly
-implemented or abstract types. There may be actions that are only implemented by
-some concrete types (e.g., not all auth methods will support a `change-password`
-action), but the permissions model still defines these at the abstract level.
-This helps keep the overall system relatively simple and predictable.
+In many cases, `output_fields` will not need to be set directly. However, an
+example in the form of some history helps provide some context as to when this
+might be useful.
 
-## Scopes are Permission Boundaries
+In Boundary 0.2.0, we restricted the set of fields returned for some `list`
+calls (those on the `scopes` and `auth-methods` collections) to a small set if
+the user was the anonymous user `u_anon`. Although default behavior in Boundary
+was to display all resource fields when listing, because the default was also to
+allow anonymous access to perform `list` on scopes and auth methods (in order to
+discover auth methods and then authenticate to them), returning all
+configuration information for all scopes and auth methods felt like it was
+publicly disclosing more information than users might be desire.
```

Comment:
```suggestion
publicly disclosing more information to users than might be desire.
```

Reply:
Fixing it myself since there's another typo on that line, but thanks for the pointer!